app/assets/stylesheets/activity-menu.scss

```diff
@@ -16,6 +16,11 @@
 }
 
 .activity-nav-mod {
+  &.footer-nav {
+    .ui-block-2 {
+      width: 769px;
+    }
+  }
```

Comment:
Ethan is going to look at whether this is necessary.
Since some themes modify the site-width class, it is possible that this hard coded width would not look right in those cases. But it seems to look fine when testing.